0200-mapping.Rmd

```diff
@@ -0,0 +1,158 @@
+  
+```{r mergin-projects, fig.cap = 'Viewing New Graph's Mergin projects',eval=T}
+knitr::include_graphics("fig/mapping/mergin_projects.png")
+```
+
+  
+  3. Right click the project name and select "Download". 
+  
+  4. Save the project somewhere appropriate. For New Graph employees this should be `Projects/gis`.
+  
+  5. Double click or right click to open the project. 
+  
+
+### Common actions
+
+Below are some common actions we use in QGIS.
+
+  1. We use themes to simplifying viewing projects. You can select a theme by clicking the eye icon in Layers Panel. 
+  
+```{r themes, fig.cap = 'How to finding features in QGIS',eval=T}
+knitr::include_graphics("fig/mapping/themes.png")
+```
+  
+  2. Finding a feature such as a culvert, stream, or else anything on the map. Use `Select feature by value`. Then input your search term and select `zoom to feature`, for example looking for the site `15600301`. 
+  
+```{r find-feature, fig.cap = 'How to finding features in QGIS',eval=T}
+knitr::include_graphics("fig/mapping/finding_feature.png")
+knitr::include_graphics("fig/mapping/site_search.png")
+```
+
+  2. Getting information about a feature ("Identify Feature"). Make sure you have the layer selected, then Select the "Identify Features" tool and click on the feature you want to know more about.
+
+```{r identify-feature, fig.cap = 'How to see feature info in QGIS',eval=T}
+knitr::include_graphics("fig/mapping/identify_feature.png")
+```
+ 
+  3. Copy information from a feature. Select the row then hover your cursor over the text you want to copy then `cmd+c` to copy.
+
+```{r copy-feature, fig.cap = 'How to copy information from a features in QGIS',eval=T}
+knitr::include_graphics("fig/mapping/copy_feature_info.png")
+```
+
+  4. Creating maps with `Projects/print layout`. This is where you can create a map that can be exported as an image or pdf. Below is an example of a map layout. Check out [this video](https://www.youtube.com/watch?v=OlDGEVhpmo0) for how to make maps.
+
+```{r qgis-maps, fig.cap = 'How to create and export maps in QGIS',eval=T}
+knitr::include_graphics("fig/mapping/qgis_maps.png")
+```
+
+
+### Editing a Project
+
+- Sync the project [following these instructions](https://merginmaps.com/docs/manage/synchronisation/#how-to-synchronise-changes-in-mergin-maps) before making any edits to make sure you have the most up to date version of the project. Others may have edited the project since you last opened it.
+
+- Make sure to sync the project [following these instructions](https://merginmaps.com/docs/manage/synchronisation/#how-to-synchronise-changes-in-mergin-maps) after making any edits.
+
+
+### Using Mergin on a phone
+
+- We find that Mergin works much better (faster rendering) with *airplane* mode on.    
```

Comment:
belongs at the top.  note that the projects should be downloaded within reach of a good wifi collection and that phones will need to have at least 2gb or more to download projects to phone (check Skeena, peace and fraser_2024 (about to make) sizes .